pkg/plugins/datasource.go

```diff
@@ -45,6 +45,8 @@ func DataSources(l logger.Interface, s schema.Stage, fs vfs.FS, console Console)
 			AvailableProviders = append(AvailableProviders, prv.NewDigitalOcean())
 		case dSProviders == "metaldata":
 			AvailableProviders = append(AvailableProviders, prv.NewMetalData())
+		case dSProviders == "vmware":
+			AvailableProviders = append(AvailableProviders, prv.NewVMware())
```

Comment:
`vmware` was added to linuxkit so I just included it after re-basing my previous fork